cds_ils/cli.py

```diff
@@ -31,26 +32,35 @@ def fixtures():
 @with_appcontext
 def pages():
     """Register CDS static pages."""
+    CURRENT_DIR = pathlib.Path(__file__).parent.absolute()
```

Comment:
there is slightly more "elegant" way to do it
https://github.com/CERNDocumentServer/cds-videos/blob/cdslabs_qa/cds/modules/fixtures/cli.py#L228f